stories/SidePanel.stories.js

```diff
@@ -27,6 +27,11 @@ import { useEffect, useState } from 'react';
 export default {
     title: 'Components/SidePanel',
     component: SidePanel,
+    args: {
```

Comment:
This is not needed storybook will pick up default values from the component based on the default value for the parameters. Same applies to the other story files.

Reply:
Sure, removed the change for this story file and other story files.